business_objects/user_activity.py

```diff
@@ -35,6 +41,28 @@ def get_all_user_activity() -> List[Any]:
     return general.execute_all(query)
 
 
+def update_last_interaction(user_id: str) -> None:
+    user_item = user.get(user_id)
+    user_item.last_interaction = sql.func.now()
+    general.commit()
+
+
+def get_active_users_in_range(
+    last_interaction_range: datetime, order_by_interaction: bool
+) -> models.User:
+    query = session.query(models.User)
+
+    if last_interaction_range:
+        query = session.query(models.User).filter(
+            models.User.last_interaction >= (last_interaction_range),
+        )
+
+    if order_by_interaction:
+        query = query.order_by(models.User.last_interaction.asc())
```

Comment:
might be wrong but i thought this would be desc() - newest first in the list

Reply:
I think it is like this -> the first one is the newest with that implementation

Reply:
This translates into (more or less) this db query:
![image](https://user-images.githubusercontent.com/91723236/222174588-dcd27c70-0044-46e3-a206-a7c6e4960e35.png)

which lists the oldest first